.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,79 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "main" ]
+  schedule:
+    - cron: '37 9 * * 6'
```

Comment:
Saturdays at 9:37 AM UTC is awfully specific

Reply:
Yea, when you have GitHub create the workflow it seems to randomly select a random weekly time. Probably to prevent everyone from hammering their CodeQL runners at the same time every week.

I just created another workflow now and it gave me `- cron: '28 13 * * 2'`

Reply:
Ah that makes sense! Well I just wanted to see if the workflow would succeed (it did), feel free to take over this PR or start a new one since they're your changes.